README.md

```diff
@@ -27,6 +29,10 @@ You will need the following services available:
 * Zookeeper 3.4.6
 * Redis
 
+You will also need:
```

Comment:
I assume this is required to run bundle install before you run `rake configure`. If so can you put this above the "You will need the following services available"?